traceur/src/main/java/com/tspoon/traceur/TraceurException.java

```diff
@@ -80,12 +108,8 @@ private boolean filterLine(final StackTraceElement element) {
         final String className = element.getClassName();
 
         // Remove references to Traceur & RxJavaPlugins
-        if (className.contains(".Traceur")
+        return !(className.contains(".Traceur")
```

Comment:
I want to make a pr to the library, so I think that change should be in another pr. makes sense?